python/utils/mtx2csv.py

```diff
@@ -0,0 +1,46 @@
+import os
+import time
+from scipy.io import mmread
+import argparse
+
+
+parser = argparse.ArgumentParser(description='Convert the sparsity pattern \
+                                 of a MatrixMarket file into a CSV file. \
+                                 Each directed edge is explicitely stored, \
+                                 edges are unsorted, IDs are 0-based.')
+parser.add_argument('file', type=str,
+                    help='Path to the MatrixMarket file')
+parser.add_argument('--csv_separator_name', type=str, default="space",
```

Comment:
You can add another parameter `choices=["space","tab","comma"]` and the parser will enforce it.

Reply:
addressed in f3a16b4